Library/Formula/logstash.rb

```diff
@@ -6,9 +6,22 @@ class Logstash < Formula
 
   bottle :unneeded
 
+  head do
+    url "https://github.com/elastic/logstash.git"
+    depends_on :java => "1.8"
+  end
+
   depends_on :java => "1.7+"
 
   def install
+    if build.head?
+      # Build the package from source
+      system "rake", "artifact:tar"
+      # Extract the package to the current directory
+      targz = Dir["build/logstash-*.tar.gz"].first
+      system "tar", "-xf", targz
```

Comment:
@mikemcquaid Thanks for the suggestion. I've pushed 802c57b247056ba081a8f4d07674d554aa011e9f.